css/main.css

```diff
@@ -179,9 +179,3 @@ a:hover {
 .hover-opacity:hover{
 	opacity: 1;
 }
-.fa-facebook{
-	color: #4267B2;
-}
-.fa-link{
-	color: #34A853;
-}

```

Comment:
I'm guessing these are not being used anywhere for you to remove them?

Reply:
footer icons are still receiving a class that colors them in the footer template:
<i class="fa icon-grey fa-{{ link.icon }} text-{{ link.iconColor }}"></i>

there are icons affected by our change are under Meet Our Team, where we feel they look better without color. happy to make them colored again though if you like, but we'll do this by applying a secondary color class like above instead of applying color directly to the icon class, to maintain reusability of said icons throughout the site

Reply:
Before:
![Screen Shot 2020-06-11 at 15 53 16](https://user-images.githubusercontent.com/33669465/84446997-b892f080-abfb-11ea-8774-8eb353c61d00.png)

Reply:
After:
<img width="795" alt="without color" src="https://user-images.githubusercontent.com/49009626/84447050-e2e4ae00-abfb-11ea-9aa6-d307c60cc080.png">